working/2364 - primary constructors/feature-specification.md

```diff
@@ -382,6 +388,46 @@ avoiding the duplication of instance variable names and types) even in the
 case where some parameters should not introduce any instance variables (so
 they are just "normal" parameters).
 
+With a primary header constructor, the formal parameters in the header are
+introduced into a new scope. This means that the parameters whose name is
+not introduced by any nested scope (e.g., the class body scope) is in scope
```

Comment:
"is" -> "are" in both places. Or "the parameters" -> "a parameter".

Reply:
Done.